worklets/Overview.bs

```diff
@@ -262,49 +265,83 @@ is empty; it is populated when the user agent chooses to create its {{WorkletGlo
 A {{Worklet}} has a list of the <dfn>worklet's resolved module URLs</dfn>. Initially this list is
 empty; it is populated when module scripts resolved.
 
+A {{Worklet}} has a <dfn>fetched modules map</dfn>. This is a map of <a>fetch</a> requests to values
```

Comment:
Changed to "module responses map". I've still got "worklet's list of resolved module URLs" at the moment as the load ordering when initializing the worklet is still important.

Could change this to just an "ordered map"?